changelogs/fragments/4393-keycloak_client_rolemapping_docs.yml

```diff
@@ -0,0 +1,4 @@
+---
+minor_changes:
```

Comment:
Could you please remove the changelog fragment? These are not needed for docs-only changes. Thanks.